aws/multi_region.go

```diff
@@ -100,6 +100,33 @@ func CloudWatchRegionsMatrix(ctx context.Context, d *plugin.QueryData) []map[str
 	return SupportedRegionMatrixWithExclusions(AWS_MONITORING_SERVICE_ID, []string{})(ctx, d)
 }
 
+func S3TablesRegionsMatrix(ctx context.Context, d *plugin.QueryData) []map[string]interface{} {
+
+	commonColumnData, err := getCommonColumns(ctx, d, nil)
+	if err != nil {
+		plugin.Logger(ctx).Error("S3TablesRegionsMatrix", "connection_name", d.Connection.Name, "unable to get partition name", err)
```

Comment:
Is the error message correct? In the above function we are trying to get the common columns data

Reply:
Updated to `plugin.Logger(ctx).Error("S3TablesRegionsMatrix", "getCommonColumns", "unable to get common column info", err)`